config/local/gateway-service.yml

```diff
@@ -8,12 +8,12 @@ apiml:
         discoveryServiceUrls: https://localhost:10011/eureka/
     security:
         auth:
-            provider: dummy
```

Comment:
I would still use `dummy` for local development.

Reply:
I have used that for testing and missed that while committing. Thank you